novelwriter/assets/themes/primer_light.conf

```diff
@@ -10,68 +10,68 @@ licenseurl = https://github.com/primer/brand/blob/main/LICENSE
 [Base]
 base    = #ffffff
 default = #1f2328
-faded   = #59636e
-red     = #fa4549
+faded   = #8c959f
+red     = #cf222e
 orange  = #e16f24
 yellow  = #d4a72c
 green   = #2da44e
-cyan    = #339D9B
+cyan    = #339d9b
 blue    = #0969da
-purple  = #a475f9
+purple  = #8557db
 
 [Project]
-root     = purple
+root     = #32383f
 folder   = orange
-file     = faded
-title    = green
-chapter  = red
-scene    = blue
+file     = #49bcb7
+title    = #116329
+chapter  = #2e934a
+scene    = #4ac26b
 note     = yellow
-active   = green
-inactive = red
-disabled = faded
+active   = #24292f
+inactive = faded
+disabled = #afb8c1
 
 [Palette]
 window          = #eaeef2
 windowtext      = default
 base            = base
-alternatebase   = #dfe4e9
+alternatebase   = #dde3e8
 text            = default
-tooltipbase     = #fae17d
-tooltiptext     = #0c1117
-button          = #f6f8fa
+tooltipbase     = #fff8c5
+tooltiptext     = default
+button          = base
 buttontext      = default
-brighttext      = faded
-highlight       = #dafbe1
-highlightedtext = #1a7f37
+brighttext      = base
+highlight       = green
+highlightedtext = base
 link            = blue
 linkvisited     = blue
-accent          = #dafbe1
+accent          = green
 
 [GUI]
-helptext  = faded
-fadedtext = faded:128
-errortext = #cf222e
+helptext  = #424a53
+fadedtext = #8c959f
+errortext = red
 
 [Syntax]
 background     = base
-text           = faded
-line           = default:32
+text           = #424a53
+line           = #eaeef2
 link           = blue
 headertext     = default
-headertag      = faded:128
-emphasis       = green
-dialog         = default
-altdialog      = default
-note           = #d4a72c
-hidden         = faded:128
+headertag      = #8c959f
+emphasis       = #2e934a
+dialog         = #000000
+altdialog      = cyan
+note           = #bf8700
+hidden         = #8c959f
 shortcode      = #e85aad
 keyword        = blue
 tag            = #218bff
-value          = #fb8f44
+value          = orange
 optional       = #54aeff
-spellcheckline = #cf222e
+spellcheckline = #fa4549
 errorline      = cyan
 replacetag     = purple
-modifier       = orange
-texthighlight  = faded:32
+modifier       = #bc4c00
+texthighlight  = #eaeef2
```

Comment:
The text highlight is practically invisible, which defeats the purpose of a highlight.